app/views/katello/api/v2/sync_plans/show.json.rabl

```diff
@@ -2,7 +2,7 @@ object @resource
 
 attributes :id, :organization_id
 attributes :name, :description
-attributes :interval, :next_sync
+attributes :interval, :next_sync, :sync_date_utc
```

Comment:
Oh, I see.  Well I think we need to figure out a way to say this in the UI because the assumption would be local time zone and we will get bug reports filed for the time being incorrect.